### PR TITLE
US2B Getting the current building as first predictions in the search bar

### LINKED ISSUE #14: US2B: As a user, I want to select my current building as a starting point.
In addition to the directions, the current building location is the default start point

### PR DESCRIPTION
**Pull request by sebhani**

### CODE REVIEW

File: components/addresses/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      this.setState({
+        coordinates: newCoordinates
+      });
+      this.props.getCoordinates(newCoordinates);
+    };
+
+    render() {
+      return (
+        <View style={styles.searchContainer}>
+          <SearchBar updateRegion={this.updateRegion} urCurentLocation={this.state.value} hideMenu={this.state.hide} />
```
Comment (eufekt) [inline components/addresses/index.js]: is this ur for "your" ? it should be stated so

Comment (sebhani) [reply components/addresses/index.js]: This part isn't related to this user story. I think @t98b wrote it.

**Review by mohhef — CHANGES_REQUESTED** [review]
Appears in the main application search bar as expected

But some other stuff still needs to be done:
- The building suggestion doesn't appear as a first choice in any of the search bars when you click on the set path button

**Comment by sebhani** [comment]
@mohhef Thank you for pointing out the bug. The bug appeared because I forgot to pass currentBuildingAddress state from Home component to Addresses Component.

**Review by gisellemartel — CHANGES_REQUESTED** [review]
Screen Shot 2020-03-22 at 5 09 02 AM
Please remove console.log() before merging. Also please merge master into this branch so we can review your changes with the latest state of the app.

Besides requested changes feature works well!

**Comment by sebhani** [comment]
@gisellemartel I removed console.log statements. Also, @trevornag merged the master into this branch two days ago.

**Comment by trevornag** [comment]
@sebhani Master changed yesterday. So I new re-merge is necessary to resolve the conflicts

**Comment by sebhani** [comment]
@trevornag, @gisellemartel I merged the master into this branch and it works fine. To test US2B functionality, please refer to the following doc:

[US2B.pdf](https://github.com/gisellemartel/CONPASS/files/4366469/US2B.pdf)

**Comment by eufekt** [comment]
Many linter issues related to single-quote string and others. Please fix. Add the eslint.json file to main directory and enable vs code es-lint.

**Comment by sebhani** [comment]
@Nichita-Hariton Linter-related issues are fixed.

**Review by TalalBazerbachi — APPROVED** [review]
The functionality seems to be working fine

**Review by eufekt — APPROVED** [review]
functionality works

**Review by mohhef — APPROVED** [review]
The functionality works fine.

**Review by gisellemartel — COMMENTED** [review]
Thank you for addressing the requested changes. Good to go!

**Review by gisellemartel — APPROVED** [review]

**Pull request merged**